superset-frontend/src/explore/components/ExploreAdditionalActionsMenu/ExploreAdditionalActionsMenu.test.jsx

```diff
@@ -43,12 +48,15 @@ describe('ExploreAdditionalActionsMenu', () => {
     ).toBe(true);
   });
   it('renders a dropdown with 3 items', () => {
-    const wrapper = mount(<ExploreAdditionalActionsMenu {...defaultProps} />, {
-      wrappingComponent: ThemeProvider,
-      wrappingComponentProps: {
-        theme: supersetTheme,
+    const wrapper = mount(
+      <ExploreAdditionalActionsMenu store={store} {...defaultProps} />,
+      {
+        wrappingComponent: ThemeProvider,
+        wrappingComponentProps: {
+          theme: supersetTheme,
+        },
```

Comment:
Could use `styledMount` here and skip the `supersetTheme` / `ThemeProvider` bits, if we wanna do a little extra sweeping up.